README.md

```diff
@@ -0,0 +1,10 @@
+## jni_gen
```

Comment:
Please add a build badge here as well.

Reply:
Should I delete the one in the jni_gen directory?

Reply:
Yes lets do that, the protobuf repo also doesn't have a build badge for the individual packages.

Reply:
Because the workflow runs on whole repo anyway?

Reply:
Exactly https://github.com/google/protobuf.dart/blob/master/.github/workflows/dart.yml